google_photos_takeout_helper/__main__.py

```diff
@@ -550,7 +560,7 @@ def fix_metadata(file: Path):
             s_date_from_folder_files.append(str(file.resolve()))
             return True
         else:
-            print('WARNING! There was literally no option to set date!!!')
+            logger.info(f'WARNING! There was literally no option to set date on {file}')
```

Comment:
Shouldn't this be a warning?

```suggestion
            logger.warning(f'There was literally no option to set date on {file}')
```

Reply:
The language suggests WARNING but I don't know if it is really more than just INFO.
This is code that has not been suggested for a pull yet.